src/Root.tsx

```diff
@@ -159,7 +160,7 @@ export const Root: FC = () => {
       },
       {
         group: false,
-        icon: HiOutlineX,
+        icon: AiOutlineMinus,
```

Comment:
I suggest using the `GrInProgress` icon, it is more significant.

Reply:
done